styles/main.css

```diff
@@ -115,26 +114,30 @@ figure img{
 	
 	text-indent: 3em; 
 }
-.useful-links {
+
+#useful-links {
 	grid-column: span 2;
 }
 
 .width-wrap{
 	box-sizing: border-box;
 	border: 1px solid rgba(0,0,0.3);
 	margin:0 auto;
-    height: 100%;
```

Comment:
Since the width-wrap element is a flex item and uses the 'stretch' alignment by default, there is no need for this rule